tools/pipelines/deploy-docs.yml

```diff
@@ -0,0 +1,266 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# deploy-docs pipeline
+
+name: $(Build.BuildId)
+
+parameters:
+  - name: deployEnvironment
+    displayName: Static web app environment to deploy to
+    type: string
+    default: new
+    values:
+      - new
```

Comment:
What are these environments? I know this is an existing variable, but I'm curious if this is still needed. @tylerbutler do you know?